hoodie-client/src/main/java/com/uber/hoodie/io/compact/HoodieCompactor.java

```diff
@@ -34,17 +32,31 @@
 
   /**
    * Compact the delta files with the data files
+   *
+   * @deprecated : Will be removed in next PR
```

Comment:
This comment is not needed here. No one is using this anyways, just leave it here with @deprecated and remove in the  next PR

Reply:
This method is getting removed in subsequent PR